### PR TITLE
Feature/users

### PR DESCRIPTION
**Pull request by j3b1d1ho0**

### CODE REVIEW

File: app.ts [hunk 1]
```diff
@@ -2,32 +2,94 @@ import * as express from 'express';
...
+      admin.save();
+  });
+});
+
+
 // view engine setup
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'ejs');
 
+
```
Comment (bamtron5) [inline app.ts]: np: extra space

**Review by JordanTreDaniel — APPROVED** [review]
We need to re-instate the logout functionality-linked view elements. Like a log-in button. Show current user's info on the nav bar.

**Comment by JordanTreDaniel** [comment]
[Add OAuth's](https://trello.com/c/l8vQKz9L/7-add-oauth-s)

**Comment by JordanTreDaniel** [comment]
[Users - 5](https://trello.com/c/pwZlPR57/4-users-5)

**Pull request merged**